docs: adjust callOnce fn description (#31747)
nuxt/nuxt: docs/3.api/3.utils/call-once.md

## Parameters

- `key`: A unique key ensuring that the code is run once. If you do not provide a key, then a key that is unique to the file and line number of the instance of `callOnce` will be generated for you.

> - `fn`: The function to run once. This function could be async or not.

Reviewer: ```suggestion
- `fn`: The function to run once. It can be asynchronous.
```
